### PR TITLE
Add Plastiq

### PR DESCRIPTION
**Pull request by brosnt**

### CODE REVIEW

File: _data/payments.yml [hunk 1]
```diff
@@ -127,6 +127,13 @@ websites:
       software: Yes
       doc: https://docs.payza.eu/docs/personal-manual-advanced-security#section-how-do-i-set-up-2-factor-authentication
 
+    - name: Plastiq
+      url: https://www.plastiq.com/
+      img: plastiq.png
+      tfa: Yes
+      sms: Yes
+      doc: https://help.plastiq.com/hc/en-us/articles/360006794893-How-do-I-enable-2-Step-Authentication-on-my-account-
```
Comment (Carlgo11) [inline _data/payments.yml]: Could you change the doc url to `https://help.plastiq.com/hc/en-us/articles/360006794893`?

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by mxxcon — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Review by Carlgo11 — COMMENTED** [review]
Just to be sure, you realize that we display the logos in a rounded format and that the circle of the Q won't be displayed, right?

**Comment by brosnt** [comment]
Re: Logo in rounded format - yup, that is fine!

**Comment by Carlgo11** [comment]
Alright.

**Pull request merged**